circle.yml

```diff
@@ -7,7 +7,6 @@ test:
     - npm run eslint
     - npm run flow -- --quiet
   override:
-    - npm run test:quiet
+    - npm run test:quiet -- --coverage --runInBand
```

Comment:
`runInBand` disables the parallelism: we don't need it in CircleCI because we have only 1 proc, and this takes too much memory. This is not slower in CircleCI with the flag.